korman/exporter/mesh.py

```diff
@@ -178,7 +178,7 @@ def __exit__(self, type, value, traceback):
             for cached_mod in override["modifiers"]:
                 mod = bo.modifiers.new(cached_mod["name"], cached_mod["type"])
                 for key, value in cached_mod.items():
-                    if key in {"name", "type"}:
+                    if key in {"name", "type"} or (cached_mod["type"] == "DECIMATE" and key=="face_count"): # Decimate attribute face_count is read-only
```

Comment:
I saw this recently when making a sample Age for ametist and was wondering what was going on. I think we can make this fix a little more flexible by making a tuple of `(cached_mod["type"], key)` and comparing that against a sequence of known read-only attributes... In other words:

```python
readonly_attribs = {("DECIMATE", "face_count"),}
# ... (some code here) ...
if key in {"name", "type"} or (cached_mod["type"], key) in readonly_attribs:
    continue
```

That way if we find any more read-only attributes like `face_count`, it's fairly trivial to add them.

Reply:
I like the tupling idea. I'll make an update. I'll go get a git GUI.

Reply:
Getting a git GUI turns out to be a problem for my old PC. Oh, well.

Reply:
I have updated per suggestion.